---
float64[6] out
